yarn-project/aztec/src/cli/cmds/start_node.ts

```diff
@@ -25,6 +22,8 @@ export const startNode = async (
   signalHandlers: (() => Promise<void>)[],
   userLog: LogFn,
 ): Promise<ServerList> => {
+  initTelemetryClient();
```

Comment:
Manually set up the global client here. We could also potentially replace this function call with passing `--import` or `--load` when starting the binary (similar to how ts-node works), e.g.:

```
node --import @aztec/telemetry-client/register some-script-that-starts-aztec
```